policies/pol/sample_arista_tp.pol

```diff
@@ -23,6 +23,16 @@ term wonky-prots-except {
   action:: accept
 }
 
+term accept-traceroute-exc {
```

Comment:
There is a file, `sample_arista_tp_yaml.yaml` that contains this same example but in YAML. Can you please update that file also?